File: 프로그래머스/unrated/161990. 바탕화면 정리/바탕화면 정리.java
class Solution {
    public int[] solution(String[] wallpaper) {
        int[] answer = {};
        int lux = 50, luy = 50, rdx = 0, rdy = 0;
        for(int i=0; i<wallpaper.length; i++){
            char[] row = wallpaper[i].toCharArray();
            for(int j=0; j<row.length; j++){
                if(row[j] == '#'){
                    if(lux > i) lux = i;
                    if(luy > j) luy = j;
                    if(rdx <= i) rdx = i+1;
                    if(rdy <= j) rdy = j+1;
                }
            }
        }
        return answer = new int[]{lux, luy, rdx, rdy};
    }
}

File: 프로그래머스/unrated/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    var answer = [50, 50, 0, 0];
    for(var i=0; i<wallpaper.length; i++){
        let tmp = wallpaper[i].indexOf('#');
        if(tmp == -1) continue;
        answer[0] = Math.min(answer[0], i);
        answer[1] = Math.min(answer[1], tmp);
        
        tmp = wallpaper[i].lastIndexOf('#');
        answer[2] = Math.max(answer[2], i+1);
        answer[3] = Math.max(answer[3], tmp+1);
    }
    return answer;
}